orangecontrib/timeseries/widgets/owdifference.py

```diff
@@ -17,32 +20,49 @@ class OWDifference(widget.OWWidget):
                   '1st or 2nd order discrete difference along its values. '
     icon = 'icons/Difference.svg'
     priority = 570
+    keywords = ['difference', 'derivative', 'quotient', 'percent change']
 
     class Inputs:
         time_series = Input("Time series", Table)
 
     class Outputs:
         time_series = Output("Time series", Timeseries)
 
+    class Operation(Enum):
+        DIFF = 0
+        QUOT = 1
+        PERC = 2
+
     want_main_area = False
     resizing_enabled = False
+    operations = ['Difference', 'Quotient', 'Percentage change']
 
+    chosen_operation = settings.Setting('Difference')
```

Comment:
So why the enum then? You really only need one string enum (`Operation`). :sweat_smile:

Reply:
At this point, i had given up on the enum. It seems i pressed undo one or two too many times and the definition was back in the code. I assume what you mean when you say string enum, is what is now in the latest iteration.